### PR TITLE
Feature/train trackmpnn on bdd

### PR DESCRIPTION
**Pull request by mez**

### CODE REVIEW

**Review by siddheshmhatre — APPROVED** [review]

File: configs/trackMPNN_detector/libra_cascade_rcnn_x101_32x4d_fpn_dconv_c3-c5_1x_coco_stage1.py [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+    warmup_ratio=0.001,
+    step=[8, 11])
+total_epochs = 15
+checkpoint_config = dict(interval=1)
+log_config = dict(interval=50, hooks=[dict(type='TextLoggerHook')])
+dist_params = dict(backend='nccl')
+log_level = 'INFO'
+load_from = None
+resume_from = None
+workflow = [('train', 1)]
```
Comment (mez) [inline configs/trackMPNN_detector/libra_cascade_rcnn_x101_32x4d_fpn_dconv_c3-c5_1x_coco_stage1.py:347]: @arangesh is this on purpose?

Comment (arangesh) [reply configs/trackMPNN_detector/libra_cascade_rcnn_x101_32x4d_fpn_dconv_c3-c5_1x_coco_stage1.py:347]: Yes, what exactly are you referring to though?

Comment (mez) [reply configs/trackMPNN_detector/libra_cascade_rcnn_x101_32x4d_fpn_dconv_c3-c5_1x_coco_stage1.py:347]: workflow doesnt include a `('val', 1)`

Comment (arangesh) [reply configs/trackMPNN_detector/libra_cascade_rcnn_x101_32x4d_fpn_dconv_c3-c5_1x_coco_stage1.py:347]: Yeah, I realized that it calculates the val metrics by default after every epoch. When you include `('val', 1)` in the workflow, it additionally calculates the validation losses too. It takes a lot of time so I decided to leave it out.

Comment (arangesh) [reply configs/trackMPNN_detector/libra_cascade_rcnn_x101_32x4d_fpn_dconv_c3-c5_1x_coco_stage1.py:347]: You can check the log files to see the validation metrics after each epoch.

**Pull request merged**